src/page/map.overview.page.tsx

```diff
@@ -231,11 +229,11 @@ const MapOverviewPage: React.FC<IMapOverviewPageProps> = (props) => {
 
     useEffect(() => {
         const payload = resolvePayload(props, state);
-        if (payload) {
+        if (undefined !== payload?.latitude && undefined !== payload.longitude ) {
             fetchMarkers(payload)
         }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [state.coordinates.latitude, state.coordinates.longitude]);
+    }, [state.coordinates, state.coordinates.latitude, state.coordinates.longitude]);
```

Comment:
from a logic, this does look like, `state.coordinates` should suffice, as it always injected as hard copy, e.g new reference will be passing update check in all cases :)